### ISSUE TITLE
UnusedPrivateElements: False positive on static private property

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#232]
Following code produces false positive _`Class Foo contains unused property $foo.`_:
```php
<?php

class Foo
{
    /** @var int[] */
    static private $foo = [123];

    public function __construct()
    {
        echo self::$foo[0];
    }
}
```

After swapping the modifiers (`private static`) the error disappears.

CS 4.1.0

**Comment by kukulich** [comment]
Thank you for your report.

It should be fixed in https://github.com/slevomat/coding-standard/commit/2aa0a398f9f459bb6a00f392befefee543bdb183
